app/scripts/modules/core/src/reactShims/ngReact.ts

```diff
@@ -42,26 +46,27 @@ export class NgReactInjector extends ReactInject {
-  public InsightLayout: React.ComponentClass<IInsightLayoutProps>                             = angular2react('insightLayout', new InsightLayoutComponent(), this.$injectorProxy) as any;
-  public InstanceArchetypeSelector: React.ComponentClass<IInstanceArchetypeSelectorProps>     = angular2react('v2InstanceArchetypeSelector', new V2InstanceArchetypeSelector(), this.$injectorProxy) as any;
-  public InstanceTypeSelector: React.ComponentClass<IInstanceTypeSelectorProps>               = angular2react('v2InstanceTypeSelector', new V2InstanceTypeSelector(), this.$injectorProxy);
-  public LegacySpinner: React.ComponentClass<ILegacySpinnerProps>                             = angular2react('spinnerWrapper', new SpinnerWrapperComponent(), this.$injectorProxy) as any;
-  public NumberList: React.ComponentClass<INumberListProps>                                   = angular2react('numberListWrapper', new NumberListWrapperComponent(), this.$injectorProxy) as any;
-  public RunningTasksTag: React.ComponentClass<IRunningTasksTagProps>                         = angular2react('runningTasksTag', { bindings: runningTasksTagBindings }, this.$injectorProxy) as any;
-  public StageSummaryWrapper: React.ComponentClass<IStageSummaryWrapperProps>                 = angular2react('stageSummary', new StageSummaryComponent(), this.$injectorProxy) as any;
-  public StepExecutionDetailsWrapper: React.ComponentClass<IStepExecutionDetailsWrapperProps> = angular2react('stepExecutionDetails', new StepExecutionDetailsComponent(), this.$injectorProxy) as any;
-  public TaskMonitorWrapper: React.ComponentClass<ITaskMonitorProps>                          = angular2react('taskMonitorWrapper', new TaskMonitorWrapperComponent(), this.$injectorProxy) as any;
-  public UserMenu: React.ComponentClass<{}>                                                   = angular2react('userMenu', {}, this.$injectorProxy) as any;
-  public ViewChangesLink: React.ComponentClass<IViewChangesLinkProps>                         = angular2react('viewChangesLinkWrapper', new ViewChangesLinkWrapper(), this.$injectorProxy) as any;
-  public ViewScalingActivitiesLink: React.ComponentClass<IViewScalingActivitiesLinkProps>     = angular2react('viewScalingActivitiesLink', new ViewScalingActivitiesLink(), this.$injectorProxy) as any;
-  public WhatsNew: React.ComponentClass<{}>                                                   = angular2react('whatsNew', {}, this.$injectorProxy) as any;
-  public TargetSelect: React.ComponentClass<ITargetSelectProps>                               = angular2react('targetSelect', new TargetSelectComponent(), this.$injectorProxy) as any;
+  public AccountRegionClusterSelector: React.ComponentClass<IAccountRegionClusterSelectorProps> = angular2react('accountRegionClusterSelectorWrapper', { bindings: accountRegionClusterSelectorBindings }, this.$injectorProxy) as any;
```

Comment:
Ah, I see you are referencing a wrapper component... did you forget to add it to this PR?

Reply:
yes, i've corrected the PR